jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/PluginConfigurationProcessor.java

```diff
@@ -42,7 +43,9 @@
 
   /**
    * Gets the value of the {@code <container><appRoot>} parameter. Throws {@link
-   * MojoExecutionException} if it is not an absolute path in Unix-style.
+   * MojoExecutionException} if it is not an absolute path in Unix-style. If this parameter is null,
```

Comment:
Rewording suggestion:

> `Gets the value of the {@code <container><appRoot>} parameter. If the parameter is {@code null}, returns {@link JavaLayerConfigurations#DEFAULT_WEB_APP_ROOT} for projects with WAR packaging or {@link JavaLayerConfigurations#DEFAULT_APP_ROOT} for other packaging.`

(Throws is removed since it is under `@throws`)

Reply:
Done